pkg/scheduler/framework/session_plugins.go

```diff
@@ -148,6 +148,11 @@ func (ssn *Session) Reclaimable(reclaimer *api.TaskInfo, reclaimees []*api.TaskI
 				continue
 			}
 			candidates := rf(reclaimer, reclaimees)
+			// intersection will be nil if lenth is 0, don't need to do any more check
+			if len(candidates) == 0 {
+				victims = nil
```

Comment:
No need for this assignment, just break

Reply:
how about the first plugin return's candidates is not null, and other plugins's candidates is null? That will cause victims has contents and not to be nil.

Reply:
ok, l see. It's my bad